# Generated for NaoQi method ALVisionVideoInput.getGVMColorSpace
# Get the color space of the requested G.V.M.

# Name under which the G.V.M. is known from ALVideoDevice.
string id
---
# { 0 = kYuv, 9 = kYUV422, 10 = kYUV, 11 = kRGB, 12 = kHSY, 13 = kBGR } -1: can't access video source
int32 value


